tests/test_configuration/test.ini

```diff
@@ -11,49 +11,131 @@ port = %(http_port)s
 [app:main]
 use = egg:kinto
 
+# Production settings
```

Comment:
This file is not correct anymore, please update it with the changes that you've made to the other one.